spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/StatsdSink.java

```diff
@@ -13,6 +13,7 @@
 
 public class StatsdSink implements Sink {
     private final static Logger logger = LoggerFactory.getLogger(StatsdSink.class);
+    private static final String MANDATORY_PREFIX = "spark";
```

Comment:
I'd suggest renaming it to `STATIC_PREFIX` and adding a short comment regarding why we need it. Something like: static prefix is used to distinguish Spark JVM source from sources of other Java applications running in a cluster

Reply:
sure, changed as suggested